appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -735,6 +735,32 @@ Blockly.WorkspaceSvg.prototype.customContextMenu = function(menuOptions) {
   };
   menuOptions.push(disableAll);
 
+  // Show all comments
+  var showAll = {enabled: true};
+  showAll.text = Blockly.Msg.SHOW_ALL_COMMENTS;
+  showAll.callback = function() {
+    var allBlocks = Blockly.mainWorkspace.getAllBlocks();
+    Blockly.Events.setGroup(true);
+    for (var x = 0, block; block = allBlocks[x]; x++) {
+      block.comment.setVisible(true);
```

Comment:
This fails if there is a block that does not have a comment (`block.comment` will be `null`).

Reply:
Done